generate_case_list/generate_case_lists.py

```diff
@@ -194,7 +194,7 @@ def get_case_list_from_staging_file(study_dir, staging_filename, verbose):
                     break # got case ids from header, don't read the rest of the file
                 else:
                     # we know at this point one of these columns exists, so no fear of ValueError from index method
-                    id_column_index = values.index(MUTATION_CASE_ID_COLUMN_HEADER) if MUTATION_CASE_ID_COLUMN_HEADER in values else values.index(SAMPLE_ID_COLUMN_HEADER)
+                    id_column_index = values.index(MUTATION_CASE_ID_COLUMN_HEADER) if MUTATION_CASE_ID_COLUMN_HEADER in values else [x.upper() for x in values].index(SAMPLE_ID_COLUMN_HEADER)
```

Comment:
So this allows "sample_id" & "sample_ID" right? How about "sampleid"?

Reply:
This is according to the SV file format in `msk_impact_2017` study.